### PR TITLE
added HAVING statement

### PR DESCRIPTION
**Pull request by th0rb**
having is really needed for some queries

### CODE REVIEW

File: LuaQuB.lua [hunk 1]
```diff
@@ -195,6 +208,30 @@ function luaqub:where( clauses, value, joiner )
 	return self
 end
 
+function luaqub:having( clauses, value, joiner )
+	if not clauses then
+		error( "Matching clauses to where function expected" )
```
Comment (hjpotter92) [inline LuaQuB.lua]: It'd be: `to having function`

**Comment by hjpotter92** [comment]
Wow! Awesome. =)

It'd also help if you could add an example in the `tests/` directory.

**Comment by th0rb** [comment]
fixed error message and added example to tests

**Comment by hjpotter92** [comment]
Cool. Thanks!

**Comment by th0rb** [comment]
Cool!

**Pull request merged**